_aggregations/pipeline/derivative.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: default
+title: Derivative
+parent: Pipeline aggregations
+nav_order: 70
+redirect_from:
+  - /query-dsl/aggregations/pipeline-agg#derivative/
+  - /query-dsl/aggregations/pipeline/derivative/
+---
+
+# Derivative aggregations
+
+The `derivative` aggregation is a parent aggregation used to calculates first-order and second-order derivatives of each bucket of an aggregation. "First-order derivative" and "second-order derivative" are often shortened to "first derivative" and "second derivative" respectively. This page uses the shortened terms.
+
+In mathematics, the first derivative of a function is its instantaneous rate of change at any point in its domain. This is commonly represented as the slope of a tangent on a graph of the function. To learn more about derivatives, see [Wikipedia](https://en.wikipedia.org/wiki/Derivative).
```

Comment:
I would remove this description. If a user uses derivative aggregation, he must know what a derivative is.